extensions/adobe/experience/campaign/notificationSubscription.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/campaign/notificationSubscription",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Notification subscription",
+  "type": "object",
+  "meta:extensible": true,
+  "description":
+    "Adobe Campaign notification subscriptions allow endusers opt into topics that can be used for communication such as Marketing subscription lists.",
+  "definitions": {
+    "notificationSubscription": {
+      "properties": {
+        "xdm:topicID": {
+          "title": "Topic Identifier",
+          "type": "string",
+          "description":
+            "The topic that the enduser has subscribed to and opted into notifications for."
+        },
+        "xdm:type": {
+          "title": "Subscription type",
+          "type": "string",
+          "description": "The type of notification/subscription."
+        },
+        "xdm:subscriptionDate": {
```

Comment:
The subscription date is at this level, but the unsubscription date is in the unsubscription details.  Is the idea that a reader knows that a user has unsubscribed if the details element is present?  If that is true, it should be described somewhere.

Reply:
Added the following description to the subscriptionDate
          "description":
            "Date and time when the enduser has subscribed to a notification topic. An enduser is subscribed if they have a Subscription Date and the `unsubscriptiondetails/unsubscriptionDate` is empty."